server/plugin/command.go

```diff
@@ -342,22 +397,80 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	if appErr = p.CreatePost(args.ChannelId, p.BotUserID, msg); appErr != nil {
+		return fmt.Sprintf("%s error creating the public post: %s", msg, appErr.Error())
+	}
+
+	return ""
```

Comment:
I ask the previous question, because we are no longer returning any ephemeral message here